UM/Operations/AddSceneNodeOperation.py

```diff
@@ -46,3 +46,8 @@ def redo(self) -> None:
         self._node.setParent(self._parent)
         if self._selected:  # It was selected while the operation was undone. We should restore that selection.
             Selection.add(self._node)
+
+    def delete(self) -> None:
```

Comment:
Maybe rename to be consistent with the other clearing methods ? Same for others
But this method may have a slightly different purpose, in which case this is fine.

Reply:
I am on it. making it more clear to what clear-> clearingData or delete-> deletingObject mean